### PR TITLE
Entity relationships

### LINKED ISSUE #34: Entity relationships

### PR DESCRIPTION
**Pull request by tomekpiotrowski**
Fixes #34

### CODE REVIEW

File: quasar_entities/src/ledger.rs [hunk 1]
```diff
@@ -24,12 +26,32 @@ pub struct Model {
...
+        Relation::Account.def()
+    }
+}
 
 impl ActiveModelBehavior for ActiveModel {}
 
 #[ComplexObject]
-impl Model {}
+impl Model {
+    pub async fn accounts<'ctx>(&self, ctx: &Context<'ctx>) -> Result<Vec<account::Model>, DbErr> {
```
Comment (geofmureithi) [inline quasar_entities/src/ledger.rs:48]: We might need a pagination part two for these, they shouldnt load everything.

**Review by geofmureithi — APPROVED** [review]
LGTM

File: quasar_entities/src/ledger.rs (hunk 1 above)
Comment (tomekpiotrowski) [reply quasar_entities/src/ledger.rs:48]: Yes

Comment (tomekpiotrowski) [reply quasar_entities/src/ledger.rs:48]: Will you be able to add it in your pagination PR after I merge this?

**Pull request merged**